docs/README.md

```diff
@@ -0,0 +1,9 @@
+## Generating documentation locally
+
+Run `scripts/document.sh`
+
+## Update the documentation site:
+1. Clone [mapbox-navigation-ios](https://github.com/mapbox/mapbox-navigation-ios) to a mapbox-navigation-ios-docs folder alongside your MapboxDirections.swift clone, and check out the `gh-pages` branch.
```

Comment:
This rigamarole is only necessary if we use the `gh-pages` branch. But we can [point GitHub Pages to the /docs folder](https://github.com/blog/2228-simpler-github-pages-publishing) in the master branch to streamline things further.